models/event.js

```diff
@@ -0,0 +1,83 @@
+const cozy = require('../cozyclient');
+const DOCTYPE = 'io.cozy.events';
+
+module.exports = {
+  displayName: 'Event',
+  doctype: DOCTYPE,
+  all (callback) {
+    cozy.data.defineIndex(DOCTYPE, [])
```

Comment:
The cozy-client-js stub (which uses fixture file) does not use the parameters and then it is not a very good test in this case. This won't work when using the yarn dev command which uses the real implementation of cozy-client-js